### PR TITLE
Reservatios display

### PR DESCRIPTION
**Pull request by MarcoAurelioAntonio**
#### - In this PR i have finish the card #8 for the project.

  - Te content was filled by a seed file... to do this I need to modify some things in Server project. For that I will do a [new PR](https://github.com/MarcoAurelioAntonio/Global-Academy-Server/pull/22) so you can check firt if you want. If is not necesary we can delete thi PR.
  - In this screen shot, you can see the route used, the render component, and the feched data from Server
  - The elements shows every thing is importatnt from the object.

image

### CODE REVIEW

**Comment by MarcoAurelioAntonio** [comment]
After merge #18 PR, I solve al merge conflicts with `dev`

File: src/redux/displayreservationsSlice.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import axios from 'axios';
+
+export const getAllReservationsApi = createAsyncThunk(
+  'api/getAllReservationsApi',
+  async (userId) => {
+    const response = await axios.get(`http://localhost:3000/api/v1/users/${userId}/reservations`);
+    // console.log(response.data);
+    return response.data;
+  },
+);
```
Comment (kensteph) [inline src/redux/displayreservationsSlice.js]: As we already have a reservationsSlice, no need to create a new one (displayreservationsSlice). You can move this block to the reservationsSlice.

File: src/redux/displayreservationsSlice.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+      state.userId = action.payload;
+    },
+  },
+
+  extraReducers: (builder) => {
+    builder
+      .addCase(getAllReservationsApi.pending, (state) => ({ ...state, status: 'loading' }))
+      .addCase(getAllReservationsApi.fulfilled, (state, action) => ({ ...state, status: 'succeeded', data: action.payload }))
+      .addCase(getAllReservationsApi.rejected, (state, action) => ({ ...state, status: 'failed', error: action.error.message }));
+  },
```
Comment (kensteph) [inline src/redux/displayreservationsSlice.js]: This block also.

**Review by kensteph — CHANGES_REQUESTED** [review]
Hi @MarcoAurelioAntonio 👋!
You did a great job, the reservations displayed correctly. However, please consider these changes.

File: src/redux/displayreservationsSlice.js (hunk 2 above)
Comment (kensteph) [reply src/redux/displayreservationsSlice.js]: After that, you can delete the displayreservationsSlice. Thank you

Comment (MarcoAurelioAntonio) [reply src/redux/displayreservationsSlice.js]: Hi @kensteph, ok good point 👍 , is better like you said. After I fixed, will tell you. Thanks!!

**Pull request merged**